### PR TITLE
Inserted Favicon and dynamic tab title

### PR DESCRIPTION
**Pull request by tonyco97**
- Inserted Nethvoice Favicon
- Inserted dynamic tab title

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -44,8 +45,24 @@ function MyApp({ Component, pageProps }: AppProps) {
     checkDarkTheme()
   }, [])
 
+  function capitalizeRouthPath() {
+    if (router.pathname) {
+      // Delete slash at the beginning of the path
+      const cleanRouterPath: string = router.pathname.replace(/^\/|\/$/g, '') || ''
+      // Return path with the uppercase first character
+      if (cleanRouterPath) {
+        return cleanRouterPath[0].toUpperCase() + cleanRouterPath.slice(1) + ' - Nethvoice'
```
Comment (gsanchietti) [inline pages/_app.tsx:54]: :warning: this should support rebranding: do not hard-code names :)

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: pages/_app.tsx [hunk 2]
```diff
@@ -44,8 +46,26 @@ function MyApp({ Component, pageProps }: AppProps) {
     checkDarkTheme()
   }, [])
 
+  const productName = getProductName()
+
+  function capitalizeRouthPath() {
```
Comment (andre8244) [inline pages/_app.tsx:51]: This is not localized. Do we have localized path name inside `router` object?

File: pages/_app.tsx [hunk 3]
```diff
@@ -44,8 +46,26 @@ function MyApp({ Component, pageProps }: AppProps) {
...
+        return cleanRouterPath[0].toUpperCase() + cleanRouterPath.slice(1) + ' - ' + productName
+      }
+    }
+  }
+
   return (
     <>
+      <div>
+        <Head>
+          <title>{capitalizeRouthPath()}</title>
```
Comment (andre8244) [inline pages/_app.tsx:66]: I would call this function something like `getPageTitle()`

**Review by andre8244 — CHANGES_REQUESTED** [review]

File: pages/_app.tsx (hunk 2 above)
Comment (andre8244) [reply pages/_app.tsx]: Nevermind, since `router` object doesn't contain localized path name we'll localize `pathname`

**Comment by tonyco97** [comment]
> The branding part of the title looks good to me. Is it possible also to change the favicon?

At the moment no, in the next PR i' want to implement something to make the favicon dynamic

**Review by gsanchietti — APPROVED** [review]
Let's wait Andrea :)

**Review by andre8244 — APPROVED** [review]

**Pull request merged**